### PR TITLE
Silhouette button

### PR DESCRIPTION
**Pull request by KatherineThayerMicrosoft**
#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [x] Include a change request file if publishing 
- [x] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

I added a new variant of Button called Silhouette, with an example.

#### Focus areas to test

(optional)

### CODE REVIEW

File: common/changes/silhouette-button_2017-01-18-17-58.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add Silhouette button variant",
```
Comment (mikewheaton) [inline common/changes/silhouette-button_2017-01-18-17-58.json]: Minor formatting request: "Button: Added a Silhouette variant"

**Comment by mikewheaton** [comment]
Would

**Comment by KatherineThayerMicrosoft** [comment]
I will wait on @micahgodbolt to check in #811 first.

File: common/changes/silhouette-button_2017-01-18-17-58.json (hunk 1 above)
Comment (dzearing) [inline common/changes/silhouette-button_2017-01-18-17-58.json]: Event better:

SilhouetteButton: Adding a new button variant component.

**Comment by Jahnp** [comment]
What is the Silhouette button for? How does it differ from other variants? Would appreciate a little context so users know how to think of this variation.

**Comment by micahgodbolt** [comment]
@KatherineThayerMicrosoft #811 is merged in. You can take a look at the new button component/s and see how we're now approaching each variant. Long and short is that we are doing away with buttonType prop and making each button variant into a separate, new component that is forked off of baseButton.

Feel free to give me a shout if you have any questions, or want some code review before submitting an PR.

For the time being, I'm going to close this PR as you'll probably want to start with a fresh fork of master and create a new branch, pulling in your silhouette styles.

**Comment by KatherineThayerMicrosoft** [comment]
Thanks Micah.

**Pull request closed without merging**